manala.ssh/CHANGELOG.md

```diff
@@ -5,6 +5,16 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Added
+- Config sshd tests
+
+### Changed
+- Cleanup readme
```

Comment:
I should avoid cleanup entries in the changelog.

Reply:
Agreed. Not sure that changelog entries about README updates are relevant also ^^

Reply:
👍